client/context/context.go

```diff
@@ -281,3 +192,56 @@ func (ctx CLIContext) PrintOutput(toPrint fmt.Stringer) (err error) {
 	fmt.Println(string(out))
 	return
 }
+
+// MessagesOutput respects flags while either generating a transaction
+// for later signing, or signing and broadcasting those messages in a transaction
+func (ctx *CLIContext) MessagesOutput(msgs []sdk.Msg) error {
```

Comment:
All CLI commands generating `sdk.Msg` will be output through this function. Once this PR is merged, then work on the tx output format will continue in another PR. This ensures that all output flags are respected and can be standardized throughout the codebase.